.github/workflows/publish-nightwatch.yml

```
@@ -69,16 +69,17 @@ jobs:
         run: |
           if [ ${{github.event.inputs.multienv}} = true ]
           then
-            echo "::set-output name=environments::[{\"node-version\":16}, {\"node-version\":14}, {\"node-version\":12}]"
+            echo "::set-output name=environments::[{\"framework-version\":1,\"node-version\":16}, {\"framework-version\":2,\"node-version\":16}, {\"framework-version\":1,\"node-version\":14}, {\"framework-version\":2,\"node-version\":14}, {\"framework-version\":1,\"node-version\":12}, {\"framework-version\":2,\"node-version\":12}]"
           else
-            echo "::set-output name=environments::[{\"node-version\":16}]"
+            echo "::set-output name=environments::[{\"framework-version\":2,\"node-version\":16}]"
           fi
   test:
     needs: [verify]
     strategy:
       max-parallel: 3
       fail-fast: false
       matrix:
+        framework-version: [1, 2]
         include: ${{fromJSON(needs.verify.outputs.environments)}}
     name: 'Test (node: ${{matrix.node-version}})'
     runs-on: ubuntu-latest
```

Here is what you've missed, to have env var for v1
```suggestion
    name: 'Test (node: ${{matrix.node-version}})'
    runs-on: ubuntu-latest
    env:
      APPLITOOLS_NIGHTWATCH_MAJOR_VERSION: ${{matrix.framework-version}}
```